packages/paste-core/components/form-pill-group/src/FormPill.tsx

```diff
@@ -73,6 +73,14 @@ export interface FormPillProps extends CompositeStateReturn {
    * @memberof FormPillProps
    */
   i18nErrorLabel?: string;
+  /**
+   * Font size of the inner text of the FormPill.
+   *
+   * @default undefined
+   * @type {BoxProps["fontSize"]}
+   * @memberof FormPillProps
+   */
+  fontSize?: BoxProps["fontSize"];
```

Comment:
question: I think I'm missing something here, why was the `fontSize` prop added? from my understanding it seems like we should be able to identify and apply the correct font sizes based on the `size` passed into FormPillGroup without any additional props (fontSize and setPillFontSize). I could totally be missing something though!

Reply:
ok seems like you're setting the larger pill size using fontSize, what if instead of passing fontSize to FormPillButton we pass size, and then FormPillButton can apply styles (height, fontSize) conditionally based on "default" | "large"?

Reply:
I actually plugged the fontSize property into a few spots including FormPillButton. I wasn't sure if there would be some instance where something other than a FormPill might go inside of it at some point.

Reply:
I can just keep the fontSize on the button if you prefer.

Reply:
yeah I think fully removing the fontSize stuff from the public API will simplify things a bit and get this in a super clean place